tests/plugins/list/backspace.html

```diff
@@ -719,3 +719,133 @@
 		</li>
 	</ol>
 </textarea>
+
+<textarea id="delete_neighboring_lists1">
+	<ol>
+		<li>a</li>
+		<li>b</li>
+		<li>c^</li>
+	</ol>
+	<ul>
+		<li>1</li>
+		<li>2</li>
+		<li>3</li>
+	</ul>
+	=>
+	<ol>
+		<li>a</li>
+		<li>b</li>
+		<li>c^</li>
+		<li>1</li>
+		<li>2</li>
+		<li>3</li>
+	</ol>
+</textarea>
```

Comment:
cliffnotes version:  ^ is where the caret is initially placed, the `=>` indicates before vs. after